sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -117,14 +119,11 @@ AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET
 
 Use the returned token credential to authenticate the client:
 ```python
-from azure.identity import DefaultAzureCredential
 from azure.ai.textanalytics import TextAnalyticsClient
-token_credential = DefaultAzureCredential()
+from azure.identity import DefaultAzureCredential
 
-text_analytics_client = TextAnalyticsClient(
-    endpoint="https://<my-custom-subdomain>.cognitiveservices.azure.com/",
```

Comment:
this specifically needs to be a custom subdomain to use AAD. why do you want to remove it? regional endpoint won't work